packages/instrumentation-openai/src/image-wrappers.ts

```diff
@@ -0,0 +1,565 @@
+import { trace, Span, SpanKind, Attributes } from "@opentelemetry/api";
+import { SpanAttributes } from "@traceloop/ai-semantic-conventions";
+import type { ImageUploadCallback } from "./types";
+import type {
+  ImageGenerateParams,
+  ImageEditParams,
+  ImageCreateVariationParams,
+  ImagesResponse,
+} from "openai/resources/images";
+
+/**
+ * Calculate completion tokens for image generation based on OpenAI's actual token costs
+ * 
+ * Token costs based on OpenAI documentation:
+ * For gpt-image-1:     Square (1024×1024)    Portrait (1024×1536)    Landscape (1536×1024)
+ * Low                  272 tokens            408 tokens              400 tokens
+ * Medium               1056 tokens           1584 tokens             1568 tokens  
+ * High                 4160 tokens           6240 tokens             6208 tokens
+ * 
+ * For DALL-E 3:
+ * Standard             1056 tokens           1584 tokens             1568 tokens  
+ * HD                   4160 tokens           6240 tokens             6208 tokens
+ */
+function calculateImageGenerationTokens(params: any, imageCount: number): number {
+  const size = params?.size || "1024x1024";
+  const quality = params?.quality || (params?.model === "gpt-image-1" ? "high" : "standard");
+  
+  // Map quality to token costs (supports both gpt-image-1 and DALL-E 3 formats)
+  const tokenCosts: Record<string, Record<string, number>> = {
+    // gpt-image-1 quality values
+    "low": {
+      "1024x1024": 272,
+      "1024x1536": 408,
+      "1536x1024": 400,
+    },
+    "medium": {
+      "1024x1024": 1056,
+      "1024x1536": 1584,
+      "1536x1024": 1568,
+    },
+    "high": {
+      "1024x1024": 4160,
+      "1024x1536": 6240,
+      "1536x1024": 6208,
+    },
+    // DALL-E 3 quality values (same costs as medium/high above)
+    "standard": {
+      "1024x1024": 1056,
+      "1024x1536": 1584,
+      "1536x1024": 1568,
+    },
+    "hd": {
+      "1024x1024": 4160,
+      "1024x1536": 6240,
+      "1536x1024": 6208,
+    }
+  };
```

Comment:
why is it here and not in our backend? then you should just report the size and model and avoid changing it every time pricing changes